### PR TITLE
Docs: Update all local links in README files to work with npm

### PR DESCRIPTION
**Pull request by gziolo**
## Description
This is a follow-up for #12835 where we discussed updating local link in README files to work with GitHub, npm and Gutenberg handbook. Thanks to changes introduced in https://meta.trac.wordpress.org/changeset/7989 and https://meta.trac.wordpress.org/changeset/7993 by @dd32, we are now able to use absolute GitHub paths to work universally in all 3 platforms 🎉 

All urls in `/packages` folder were updated to avoid directory traversing.

### CODE REVIEW

File: packages/blocks/README.md [hunk 1]
```diff
@@ -94,8 +94,7 @@ __You will need to provide implementations both for what's to be shown in an
 editor and what's to be saved with the published content__.
 
 To eliminate redundant effort here, share common behaviors by splitting your
-code up into
-[components](../element).
+code up into [components](packages/element).
```
Comment (gziolo) [inline packages/blocks/README.md]:
```suggestion
 code up into [components](/packages/element/README.md).
```

**Comment by gziolo** [comment]
I noticed some inconistent new lines added in `blocks` README file and I fixed them in https://github.com/WordPress/gutenberg/pull/13030/commits/e8269882380f12d1eb2f2faa539af936360e5daf.

File: packages/nux/README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 See [the component's README][dot-tip-readme] for more information.
 
-[dot-tip-readme]: https://github.com/WordPress/gutenberg/tree/master/packages/nux/src/components/dot-tip/README.md
+[dot-tip-readme]: /packages/nux/src/components/dot-tip/README.md
```
Comment (dd32) [inline packages/nux/README.md]: It doesn't look like we currently handle components of packages, ie. the `dot-tip` component doesn't exist in the WordPress.org handbook. I suspect the link to the readme is intentional here?

Comment (gziolo) [reply packages/nux/README.md]: Good catch, it shouldn't probably. I will revert this line :)

**Comment by gziolo** [comment]
@dd32, I update all links that wouldn't work. I went to far with updates :)

**Review by ajitbohra — APPROVED** [review]
LGTM :+1: 	

Seems like it would be good to add a note to https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#contribute-to-the-documentation 

On adding links to docs.

**Comment by gziolo** [comment]
Once we confirm it works everywhere as we anticipate, we definitely should 😃 

Are there any Markodwn linters?

**Comment by ajitbohra** [comment]
yes, we have many options we can zero down on one to cut down formatting reviews and bring in some consistency across docs. 

https://github.com/markdownlint/markdownlint
https://github.com/remarkjs/remark-lint
https://github.com/DavidAnson/markdownlint

**Comment by ajitbohra** [comment]
@chrisvanpatten thanks for linking this was trying to recollect the same :+1:

**Pull request merged**